install.sh

```diff
@@ -219,7 +219,7 @@ old_glibc() {
 if [[ -n "${HOMEBREW_ON_LINUX-}" ]] && old_ruby && old_glibc; then
     abort "$(cat <<EOFABORT
 Homebrew requires Ruby 2.6 which was not found on your system. 
-Homebrew can use vendored Ruby but it requires Glibc version 2.13 or newer.
+Homebrew's portable Ruby requires Glibc version 2.13 or newer
```

Comment:
```suggestion
Homebrew's portable Ruby requires Glibc version 2.13 or newer.
```

Reply:
Message updated in 761430e575b0ddf83fdbe2e3d024c95d565339ee.